tools/go-cli/go-whisk-cli/main.go

```diff
@@ -20,17 +20,30 @@ import (
     "fmt"
     "os"
     "reflect"
+    "time"
 
     "../go-whisk/whisk"
     "../go-whisk-cli/commands"
 )
 
+// CLI_BUILD_TIME holds the time of the CLI build.  During production builds,
+// this value will be overwritten via the command:
+//     go build -ldflags "-X main.CLI_BUILD_TIME=nnnnn"   // nnnnn is the new timestamp
+var CLI_BUILD_TIME string = "not set"
+
 var cliDebug = os.Getenv("WSK_CLI_DEBUG")  // Useful for tracing init() code
 
 func init() {
     if len(cliDebug) > 0 {
         whisk.SetDebug(true)
     }
+
+    // Initialize a default CLI build timestamp value only if the value was not
+    // set via the go linker flags.
+    if CLI_BUILD_TIME == "not set" {
+        CLI_BUILD_TIME = time.Now().Format("2006-01-02T15:04:05-07:00")
```

Comment:
There's actually not a hardcoded timestamp.  It sure looks that way due to the way the Go time format string must be specified.  Unlike in Java, where the date format string looks something like "yyyy-MM-dd....", Go requires an actual arbitrary date and time.
http://stackoverflow.com/questions/20234104/how-to-format-current-time-using-a-yyyymmddhhmmss-format

The actual default timestamp is set to "now" - the time the init() function is called.  Yeah, I debated a bit about this default timestamp.  I wanted local builds to have a timestamp, but you are right in that it might give a false positive to the production builds